### PR TITLE
use_services

### PR DESCRIPTION
**Pull request by Pchristy03**

### CODE REVIEW

File: src/Controllers/UserController.cs [hunk 1]
```diff
@@ -1,39 +1,33 @@
...
 using magnus_backend.Models;
-using Microsoft.AspNetCore.Authorization;
+using magnus_backend.Services;
 using Microsoft.AspNetCore.Mvc;
-using MongoDB.Driver;
 
 namespace magnus_backend.Controllers;
 
-[Authorize]
+//[Authorize]
```
Comment (averyschreiner) [inline src/Controllers/UserController.cs]: shouldn't we uncomment this?

Comment (averyschreiner) [reply src/Controllers/UserController.cs]: and add `using Microsoft.AspNetCore.Authorization;`

**Comment by averyschreiner** [comment]
should we change this pr to merge into dev?

File: src/Controllers/UserController.cs (hunk 1 above)
Comment (Pchristy03) [reply src/Controllers/UserController.cs]: ope good catch, testing about got us lol

**Comment by Pchristy03** [comment]
> should we change this pr to merge into dev?

Do we need a dev version on the backend? If we merge it in it should just immediately take affect on azure because of our build process right?

**Pull request merged**